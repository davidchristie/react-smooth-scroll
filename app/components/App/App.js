import React from 'react'

import Link from '../../containers/Link'
import Menu from '../Menu'
import Section from '../Section'

export default class App extends React.Component {
  render () {
    return (
      <div>
        <Menu>
          <Link to='section1'>Section 1</Link><br />
          <Link to='section2'>Section 2</Link><br />
          <Link to='section3'>Section 3</Link><br />
          <Link to='section4'>Section 4</Link><br />
          <Link to='section5'>Section 5</Link>
        </Menu>
        <Section id='section1' style={{backgroundColor: '#96ceb4'}}>
          <h1>Section 1</h1>
        </Section>
        <Section id='section2' style={{backgroundColor: '#ffeead'}}>
          <h1>Section 2</h1>
        </Section>
        <Section id='section3' style={{backgroundColor: '#ff6f69'}}>
          <h1>Section 3</h1>
        </Section>
        <Section id='section4' style={{backgroundColor: '#ffcc5c'}}>
          <h1>Section 4</h1>
        </Section>
        <Section id='section5' style={{backgroundColor: '#88d8b0'}}>
          <h1>Section 5</h1>
        </Section>
      </div>
    )
  }
}
