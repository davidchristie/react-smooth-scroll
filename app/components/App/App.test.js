import { shallow } from 'enzyme'
import React from 'react'

import App from '../App'

describe('<App />', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<App />)
    expect(wrapper.getNodes()).toMatchSnapshot()
  })
})
