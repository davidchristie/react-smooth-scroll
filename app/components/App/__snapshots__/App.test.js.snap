// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should match snapshot 1`] = `
<div>
  <Menu>
    <Connect(Link)
      to="section1"
    >
      Section 1
    </Connect(Link)>
    <br />
    <Connect(Link)
      to="section2"
    >
      Section 2
    </Connect(Link)>
    <br />
    <Connect(Link)
      to="section3"
    >
      Section 3
    </Connect(Link)>
    <br />
    <Connect(Link)
      to="section4"
    >
      Section 4
    </Connect(Link)>
    <br />
    <Connect(Link)
      to="section5"
    >
      Section 5
    </Connect(Link)>
  </Menu>
  <Section
    id="section1"
    style={
      Object {
        "backgroundColor": "#96ceb4",
      }
    }
  >
    <h1>
      Section 1
    </h1>
  </Section>
  <Section
    id="section2"
    style={
      Object {
        "backgroundColor": "#ffeead",
      }
    }
  >
    <h1>
      Section 2
    </h1>
  </Section>
  <Section
    id="section3"
    style={
      Object {
        "backgroundColor": "#ff6f69",
      }
    }
  >
    <h1>
      Section 3
    </h1>
  </Section>
  <Section
    id="section4"
    style={
      Object {
        "backgroundColor": "#ffcc5c",
      }
    }
  >
    <h1>
      Section 4
    </h1>
  </Section>
  <Section
    id="section5"
    style={
      Object {
        "backgroundColor": "#88d8b0",
      }
    }
  >
    <h1>
      Section 5
    </h1>
  </Section>
</div>
`;
